ui/src/app/machines/views/MachineDetails/MachineTests/MachineTestsTable/MachineTestsTable.tsx

```diff
@@ -56,24 +139,37 @@ const renderActions = (
   );
 };
 
+const useVisible = () => {
```

Comment:
We have a similar (but different) hook in base hooks with the same name.. I'm not certain what name to give this, `useVisibleById` to move it there?